gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/SnapshotController.groovy

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.controllers
+
+import com.netflix.spinnaker.gate.services.SnapshotService
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@Slf4j
+@CompileStatic
+@RestController
+@RequestMapping("/snapshots")
+class SnapshotController {
+
+  @Autowired
+  SnapshotService snapshotService
+
+  @RequestMapping(value = "/{application}/{account}", method = RequestMethod.GET)
+  Map getCurrentSnapshot(@PathVariable("application") String application,
+                         @PathVariable("account") String account) {
+    snapshotService.getCurrent(application, account)
```

Comment:
I might suggest going with `/applications/{application}/snapshots/{account}`.

If we really want a `/snapshots`, it should be `/snapshots/{id}` IMO.